metrics/lib/send_results.sh

```diff
@@ -36,7 +36,7 @@ COR_DEFAULT_PATH="/usr/bin"
 COR_DEFAULT_NAME="cc-oci-runtime"
 RUNC_DEFAULT_PATH="/usr/bin"
 RUNC_DEFAULT_NAME="docker-runc"
-CC_RUNTIME_DEFAULT_PATH="/usr/local/bin"
+CC_RUNTIME_DEFAULT_PATH="/usr/bin"
```

Comment:
What about checking first if the the `cc-runtime` binary is installed in `/usr/local/bin`, using `which` or `command`. And add an `echo: Using cc-runtime from: $INSTALLED_PATH` so that the user is aware of that. If the user has not installed the runtime by running the `make install`, then he will only have the runtime installed in `/usr/bin`